app/views/comparisons/annual_change_in_gas_out_of_hours_use/_table.csv.ruby

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+render 'comparisons/shared/annual_change_in_out_of_hours_use/table'
```

Comment:
I have passed locals: local_assigns through here when doing a similar thing, which makes me wonder if I've done it right. I will check